readme.md

```diff
@@ -63,3 +63,35 @@ ecs-upload-task --file taskdefinition.yml --service hello-world-2017-05-15-10-45
 2017/11/14 18:03:48 (service hello-world-2017-05-15-10-45) has stopped 1 running tasks: (task a3e3a91b-be05-4092-bcf6-47f2075933af).
 
 ```
+
+## Creating a new multi-arch release
+
+With the uptake of M1 Macs amongst devs, we need to create builds for multiple CPU architectures to ensure compatability across all devices. This uses [`gox`](https://github.com/mitchellh/gox) to do the multi-arch builds. This can be installed by
+
+```
+go install github.com/mitchellh/gox
+```
+
+Following this:
+
+1. Check out the the commit you want to create a release for, and tag it with appropriate semver convention:
+
+```
+$ git tag vx.x.x
+$ git push --tags
+```
+
+2. Create the binaries:
+
+```
+$ make clean
+$ make release-multi-arch
```

Comment:
You should only need `make` here, since it defaults to running the first target in the Makefile. The convention is to label the first target in the Makefile `all` to make this clearer.

Reply:
Yeah, either that or just put the commands directly under the `all` target.